docs/translations-needed.md

```diff
@@ -2,7 +2,7 @@
 
 The currently available translations are in the lang dir. This table shows the completeness of those translations. Anything not listed does not exist yet, so go ahead and create it by copying `en.json`.
 
-If you add or update a translation run `grunt check-translations` to update the list and include this modified doc in the pull request.
+If you add or update a translation run `node build/translations.js` to update the list and include this modified doc in the pull request.
```

Comment:
this can actually be `npm run docs:lang`.
We need to update all the docs if they mention grunt.